web/docs/develop/architecture/plugins.md

```diff
@@ -126,4 +127,4 @@ use the [printer](#printer) plugin instead.
 The writer plugin adds a new format to print data, such as JSON (ASCII) or PCAP
 (binary).
 
-Writer plugins automatically add the subcommand `vast export <plugin name>`.
+Writer plugins automatically add the subcommand `tenzir export <plugin name>`.
```

Comment:
Writer plugins and reader plugins will go away as part of the renaming.

Reply:
I thought about this. But should we just leave this section there as a reminder to adapt the content to the connectors and formats plugins?

Reply:
Noted this down as separate TODO and will get back to it.